Makefile

```diff
@@ -15,16 +15,12 @@ ci\:diff\:from:
 	@if [ "$(shell git rev-parse --abbrev-ref HEAD)" = "master" ]; then \
 	    git --no-pager log --first-parent master --merges -n 2 --pretty=format:"%H" | tail -n 1; \
 	else \
-		echo "HEAD"; \
+		echo "remotes/origin/master"; \
 	fi
 
 .PHONY: ci\:diff\:to
 ci\:diff\:to:
-	@if [ "$(shell git rev-parse --abbrev-ref HEAD)" = "master" ]; then \
-	    echo "HEAD"; \
-	else \
-		echo "remotes/origin/master"; \
-	fi
+	@echo "HEAD";
```

Comment:
Fixed the problem that the difference does not appear in `make ci:diff` when new image.
The branch compares the master and HEAD, and the master compares the previous merge branch with the HEAD.